backend/initdb/01-schema.sql

```diff
@@ -45,7 +45,7 @@ CREATE TABLE event (
                        FOREIGN KEY (object_id) REFERENCES object(id)
 );
 
-CREATE TABLE user (
+CREATE TABLE users (
```

Comment:
"user" jest keywordem dla postgresa i generalnie odradzane jest stosowanie tego jako nazwy tabeli, [chociaż jest to możliwe](https://stackoverflow.com/a/22256451). Bez nawiasów, czyli tak jak było, wyrzucało syntax error przy tworzeniu tabeli